gpii/node_modules/flowManager/src/RevisionGetHandler.js

```diff
@@ -0,0 +1,55 @@
+/*!
+GPII GET /revision handler used by the Cloud Based Flowmanager
+
+Copyright 2020 OCAD University
+
+Licensed under the New BSD license. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the License at
+https://github.com/GPII/universal/blob/master/LICENSE.txt
+*/
+
+"use strict";
+
+var fs = require("fs"),
+    fluid = require("infusion"),
+    gpii = fluid.registerNamespace("gpii");
+
+fluid.defaults("gpii.flowManager.cloudBased.gpiiRevision.handler", {
+    gradeNames: ["kettle.request.http"],
+    invokers: {
+        handleRequest: {
+            funcName: "gpii.flowManager.cloudBased.gpiiRevision.handler.handleRequest",
+            args: ["{gpii.flowManager.cloudBased}", "{that}"]
```

Comment:
Please pass `{gpii.flowManager.cloudBased}.options.gpiiRevisionPath` as the first argument since it's the only value required from `gpii.flowManager.cloudBased` component, so the function doesn't require the entire component as an input.